typescript/packages/browser/go/browserutil/browserutil.go

```diff
@@ -10,10 +10,10 @@ import (
 	"syscall/js"
 	"time"
 
-	"github.com/0xProject/0x-mesh/browser/go/jsutil"
-	"github.com/0xProject/0x-mesh/browser/go/providerwrapper"
 	"github.com/0xProject/0x-mesh/core"
 	"github.com/0xProject/0x-mesh/orderfilter"
+	"github.com/0xProject/0x-mesh/typescript/packages/browser/go/jsutil"
```

Comment:
Importing from __typescript/something/go__ is a little awkward. If the __typescript/__ directory doesn't just contain TypeScript code, maybe we should rename it? Would you be comfortable just having a top-level __packages/__ directory?

Reply:
That works for me!